### DISCUSSION TITLE
The scripts stop working after update

### DISCUSSION THREAD

**Discussion by SkyMaggie** [thatskymod/Sky-CotL-Scripts#68] [General]
The scripts stop working after game was updated it's says the session was terminated
Please help anyone!! 🙏🙏🙏😭
![Screenshot_20240412_084947_Canvas](https://github.com/thatskymod/Sky-CotL-Scripts/assets/166769866/c72aa94f-9c3f-4a52-84ef-db6808c36eb7)

**Comment by Snoopo69** [comment]
Same here, this sucks, I don't have time to farm like 2-3 hours

**Comment by thatskymod** [comment]
# Fixed in libLangit017rc.so
The session error you mentioned has been fixed in [Canvas/Libs/libLangit017rc.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libLangit017rc.so).

Thank you for your patience while the issue was being worked on.
